apps/wizard/cli.py

```diff
@@ -18,6 +18,16 @@
 # Disable streamlit cache data API logging
 # ref: @kajarenc from https://github.com/streamlit/streamlit/issues/6620#issuecomment-1564735996
 logging.getLogger("streamlit.runtime.caching.cache_data_api").setLevel(logging.ERROR)
+# RICH-CLICK CONFIGURATION
```

Comment:
This is repeated in `apps/cli/__init__.py`. Can we put it in one place? (I'd like to reuse it too)